.travis.yml

```diff
@@ -1,29 +1,29 @@
 language: go
 
 go:
-  - 1.5
```

Comment:
I just recently fixed support for these versions.
Is there a reason why you dropped support here?
Of course that would break your changes in `before_install`